src/events/EnterPassword.java

```diff
@@ -0,0 +1,16 @@
+package events;
+
+public class EnterPassword extends SecurityEvent {
+	private static EnterPassword instance;
+
+	private EnterPassword() {
+
+	}
+
+	public static EnterPassword instance() {
+		if (instance == null) {
+			instance = new EnterPassword();
+		}
+		return instance;
+	}
+}
```

Comment:
Need a new line here

Reply:
Same answer here; I'll try and see if I have too many empty lines anywhere.